server/src/modules/campaign-monitor/campaign-monitor.service.ts

```diff
@@ -44,9 +47,11 @@ checkRequiredEnvs([
 	'CAMPAIGN_MONITOR_EMAIL_TEMPLATE_TEMP_ACCESS_END',
 	'CAMPAIGN_MONITOR_LISTS_ENDPOINT',
 	'CAMPAIGN_MONITOR_LIST_SYNC_BATCH',
+	'AVO_LIST_SYNC_BATCH',
 ]);
 
 const listSyncBatchSize = parseInt(process.env.CAMPAIGN_MONITOR_LIST_SYNC_BATCH as string, 10);
+const avoListSyncBatchSize = parseInt(process.env.AVO_LIST_SYNC_BATCH as string, 10);
```

Comment:
Let's add a sensible default in case these env vars are not set.
```suggestion
const listSyncBatchSize = parseInt((process.env.CAMPAIGN_MONITOR_LIST_SYNC_BATCH || '1000') as string, 10);
const avoListSyncBatchSize = parseInt((process.env.AVO_LIST_SYNC_BATCH || '1000') as string, 10);
```